.travis.yml

```diff
@@ -15,6 +15,9 @@ cache:
   - composer
   - $HOME/.composer/cache
 
+install:
+  - composer install --no-interaction
```

Comment:
Off the cuff, two reasons: security and parity with development environment. I was surprised to see this was even allowed with `sudo: false` TBH.